spec/string_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../string'
+
+RSpec.describe String do
+
+  describe "#numeric?" do
+    it 'determines if a certain string containes only numeric values' do
```

Comment:
A test should have one expectation. Consider using something like https://github.com/rspec/rspec-its for cases like this or extracting to separate `it` tests.